setup.py

```diff
@@ -25,7 +25,7 @@
 
     setup(
         name="regmodsm",
-        version="0.1.1",
+        version="0.2.1",
```

Comment:
Thanks @kels271828! Here I have some questions about versioning to ask you guys @kels271828, @geoffreyphipps-uw
* This PR is only a part of the feature I want to introduce to `regmodsm` should I create another branch for this change into rather than directly merge into main. After merge this PR, the code WILL break, it has to be incoordination with other PR to make it work.
* The final version will probably break the backward compatibility. Should the version be `1.0.0` or `1.0.1`? And assume we only add a feature, should the version be `0.2.1` or `0.2.0`?